File: 28-07-2018/Act.cls/notas.rb
class  Main
	attr_accessor :total, :promedio

	def initialize(total = 0, promedio = 0)
		@total=total
		@promedio=promedio
	end
	def calculateAvg
		notas = []
		puts "#{notas}"
		for i in(0..6)
			print "Ingrese nota: "
    		nota=gets.to_i
   			notas.push nota
   			self.total = notas[i].to_i + total
   			if notas[i] == 0
				self.promedio = self.total/i;
				break;
			end
			self.promedio = self.total/notas.length
		end
		puts("El total es #{total}")
		puts "El promedio es: #{promedio}"
	end
end

if __FILE__ == $0
  ma = Main.new
  ma.calculateAvg
end

File: 21-07-2018/Act.cls/01-NotasJavascript/notas.js
var nota1 =0;
var nota2 =0;
var nota3 =0;
var nota4 =0;
var nota5 =0;
var nota6 =0;
var nota7 =0;
var notas = new Array(nota1, nota2, nota3, nota4, nota5, nota6, nota7);
var total = 0;
var promedio =0;
for(let i=0; i<7;i++){
	notas[i] = prompt("Ingrese nota");
	total = total+parseFloat(notas[i]);
	if(notas[i]==0){
		promedio = total/i;
		break;
	}
	console.log("El total es: " +total);
	promedio = total/notas.length;
}
alert("El promedio es: "+promedio);


